packages/activemodel-adapter/tests/integration/active-model-serializer-test.js

```diff
@@ -341,3 +341,18 @@ test("extractErrors camelizes keys", function() {
     firstName: ["firstName not evil enough"]
   });
 });
+
+test('extractErrors extracts polymorphic records errors', function() {
+  var payload = {
+    errors: {
+      evil_minions: [
+        { name: ['required'] }
+      ]
+    }
+  };
+
+  var serializer = env.container.lookup('serializer:application');
+  serializer.extractErrors(env.store, MediocreVillain, payload, null); // id (last param) is not relevant here
+
+  ok(true);
```

Comment:
Actually, it's a bit more complicated than that. The serializer will _not_ change the key from `evil_minion` to `evilMinions` because it is looking for `evil_minion_ids`.

So there are two things that are relevant in this test:
1) No matter what you pass in, `extractErrors` should crash (eg: _not_ look for a `.type` property on an error payload)
2) If the serializer is configured to rename the `evil_minions` association to `evilMinions`, errors should also be renamed.

I updated the test again to cover (2), but I had to bring in EmbeddedRecord.